tests_integration_with_local_chain/nft_agents/test_agent_communication.py

```
@@ -33,48 +35,84 @@ def test_count_unseen_messages(local_web3: Web3, accounts: list[TestAccount]) ->
+        f"balance {get_balances(address=test_keys.bet_from_address, web3=local_web3)}"
+    )
     comm_contract = AgentCommunicationContract()
 
+    # It might be the case that initial_messages > 0 (due to ape's tests not being isolated).
     initial_messages = comm_contract.count_unseen_messages(
         agent_address=mock_agent_address, web3=local_web3
     )
-    assert initial_messages == 0  # no messages yet
 
+    # add new message
     message = zlib.compress(b"Hello there!")
```

Ruff (0.8.2)

Local variable `message` is assigned to but never used

Remove assignment to unused variable `message`

(F841)